connectivity/manifests/client-egress-l7-http-named-port.yaml

```diff
@@ -0,0 +1,39 @@
+---
+# client2 is allowed to contact one.one.one.one/ on port http-80 and the echo Pod
```

Comment:
CIDR/toFQDN destinations can't have named ports, so maybe remove that from the comment? Policy seems correct in this regard already.